dicoogle/src/main/resources/webapp/js/handlers/requestHandler.js

```diff
@@ -193,12 +193,13 @@ function saveIndexOptions(path, watcher, zip, saveThumbnail, effort, thumbnailSi
 
 }
 
-function forceIndex(uri){
+function forceIndex(uri, plugin){
   //console.log(state);
   // TODO use dicoogle client
   $.post(Endpoints.base + "/management/tasks/index",
```

Comment:
You should instead use our Dicoogle client. The [`index`](https://bioinformatics-ua.github.io/dicoogle-client-js/interfaces/_types_dicoogle_client_d_.dicoogleclient.dicoogleaccess.html#index) method already supports a list of index providers.